Assets/MixedRealityToolkit.Extensions/SpectatorView/Scripts/StateSynchronization/StateSynchronizationObserver.cs

```diff
@@ -27,6 +32,10 @@ public class StateSynchronizationObserver : Singleton<StateSynchronizationObserv
         [SerializeField]
         protected int port = 7410;
 
+        [Tooltip("The anchor prefab.")]
+        [SerializeField]
+        private GameObject anchorPrefab = null;
+
```

Comment:
It's not clear what this is used for. Is the anchor related to an origin? Or a spatial coordinate?